### ISSUE TITLE
Question about refresh time

### ISSUE THREAD

**Issue by alijaya** [webtiming/timingsrc#5]
Hi, I'm not sure if I could explain clearly what I want to ask.
So basically, I want to sync multiple device that shows "karaoke" like app.
So the app use requestAnimationFrame to do the main loop (so the refresh rate is pretty smooth).
The app already supports the play, pause, seek feature.
I'm not sure how to integrate this library to my app.

Firstly, do I need to only listen to "timeupdate" event only to refresh my app? But I'm not sure it will be smooth (or it could be compared to requestAnimationFrame). Or I just need to listen the command (like "play", "pause", etc), and for each device they do the command, and resync once a while?

Basically, I'm curious how you guys treat the HTMLMediaElement / MediaSync, because my case it's kinda like that, but I can not use the library, because MediaSync is only for HTMLMediaElement, not general enough.

I hope I explain clear enough. Thanks.

**Comment by Snarkdoof** [comment]
I would use requestAnimationFrame and use the timingObject.pos to get the current position on refresh, or put the graphics (texts) in a sequencer and have it trigger coloring or something like that. The timeupdate event is only at 5 hz, so it's mostly useful for progress bars and that kind of thing.

**Comment by alijaya** [comment]
Ah... I see... Now I kinda understand how it works... we can use timeObject.query() anytime to get current pos, so I can use requestAnimationFrame and then query the timeObject to see where am I now.
Thanks! :D

**Comment by Snarkdoof** [comment]
Exactly, no problem. :)
